src/hot-keys/RaceSpecificHotkeys/hotkeyData/human/units.ts

```diff
@@ -76,7 +76,7 @@ const units =
         ]),
         new Unit('Dragonhawk Rider', 'btndragonhawkrider', [
             ...defaultFighterAbilities,
-            Ability.Create('Aerial Shackles', 'btnmagiclariet', 'AMLS', 'E', []),
+            Ability.Create('Aerial Shackles', 'btnmagiclariet', 'amls', 'E', []),
```

Comment:
case-sensitivity matters for some reason. could possibly do a later fix to make case-insensitive.

Reply:
i think because we import them with .toLower() and then compare them.

Reply:
yeah, it's something we can fix later anyway. for now, I'd say we keep it lower case.